website/docs/api/plugins/plugin-content-blog.md

```diff
@@ -38,7 +38,7 @@ Accepted fields:
 | `routeBasePath` | `string` | `'blog'` | URL route for the blog section of your site. **DO NOT** include a trailing slash. Use `/` to put the blog at root path. |
 | `include` | `string[]` | `['**/*.{md,mdx}']` | Matching files will be included and processed. |
 | `exclude` | `string[]` | _See example configuration_ | No route will be created for matching files. |
-| `postsPerPage` | `number` | `10` | Number of posts to show per page in the listing page. |
+| `postsPerPage` | <code>number &#124; 'ALL'</code> | `10` | Number of posts to show per page in the listing page. Use `'ALL'` to display all posts on one listing page. |
```

Comment:
Are you sure this doesn't work? `number | 'ALL'`

Reply:
No it doesn't. If I could I would 🤦‍♂️

`` `number | 'ALL'` ``

![image](https://user-images.githubusercontent.com/55398995/129376522-6504b2be-db98-430d-bbf5-9db30f7d5ea9.png)

`<code>number | 'ALL'</code>`

![image](https://user-images.githubusercontent.com/55398995/129376328-c3ee70a6-d47e-4469-8dc8-20c155f39cfb.png)

The table structure is extracted prior to processing each cell

Reply:
I see thanks. I don't have much experience with MDX tables yet 😅